ruby/matcher_test.go

```diff
@@ -106,7 +106,7 @@ func TestVulnerable(t *testing.T) {
 			record: &claircore.IndexRecord{
 				Package: &claircore.Package{
 					Name:    "dependabot-omnibus",
-					Version: "0.120.0-beta2",
```

Comment:
This was an invalid semver before, so I switched this to ensure our Rubygem version sees this as valid